### PR TITLE
fix: resolved issue with incorrect date type when load data into online table

### LINKED ISSUE #1103: `Date` type value is incorrect when load data into online table
row data:
bfa650d027b33ef19cb81c560661ddc5

select from online table
688ef9decdfe63ff33a5d9ff94f58f9a

### PR DESCRIPTION
**Pull request by yabg-shuai666**
close #1103

### CODE REVIEW

File: java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ private void addRow(InternalRow record, PreparedStatement preparedStatement) thr
                     preparedStatement.setString(i + 1, record.getString(i));
                     break;
                 case Types.DATE:
-                    preparedStatement.setDate(i + 1, new Date(record.getInt(i)));
+                    preparedStatement.setDate(i + 1, new Date(record.getInt(i)*86400000l));
```
Comment (vagetablechicken) [inline java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java]: plz add comment about why

Comment (yabg-shuai666) [reply java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java]: ok!

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java [hunk 2]
```diff
@@ -110,7 +110,9 @@ private void addRow(InternalRow record, PreparedStatement preparedStatement) thr
                     preparedStatement.setString(i + 1, record.getString(i));
                     break;
                 case Types.DATE:
-                    preparedStatement.setDate(i + 1, new Date(record.getInt(i)));
+                    // the new Date() parameter is the number of milliseconds elapsed since January 1, 1970 00:00:00 GMT
+                    // where record.getInt(i) gets the number of days elapsed, and 86400000 is the number of milliseconds in a day.
+                    preparedStatement.setDate(i + 1, new Date(record.getInt(i) * 86400000l));
```
Comment (tobegit3hub) [inline java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java:115]: Please add new unit test for this type.

**Review by vagetablechicken — APPROVED** [review]

File: java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java (hunk 2 above)
Comment (vagetablechicken) [reply java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java:115]: later

**Pull request merged**